perf/benchmark/run_benchmark_job.sh

```diff
@@ -44,23 +44,33 @@ function setup_metrics() {
 }
 
 function collect_metrics() {
-  local GENERATE_GRAPH=$1
-  local PLOT_METRIC=$2
-  CSV_OUTPUT="$(mktemp /tmp/benchmark_XXXX.csv)"
+  # shellcheck disable=SC2155
+  export CSV_OUTPUT="$(mktemp /tmp/benchmark_XXXX.csv)"
   pipenv install
   pipenv run python3 fortio.py $FORTIO_CLIENT_URL --csv_output="$CSV_OUTPUT" --prometheus=$PROMETHEUS_URL \
    --csv StartTime,ActualDuration,Labels,NumThreads,ActualQPS,p50,p90,p99,cpu_mili_avg_telemetry_mixer,cpu_mili_max_telemetry_mixer,\
 mem_MB_max_telemetry_mixer,cpu_mili_avg_fortioserver_deployment_proxy,cpu_mili_max_fortioserver_deployment_proxy,\
 mem_MB_max_fortioserver_deployment_proxy,cpu_mili_avg_ingressgateway_proxy,cpu_mili_max_ingressgateway_proxy,mem_MB_max_ingressgateway_proxy
+}
 
-  if [[ "$GENERATE_GRAPH" = true ]];then
-    BENCHMARK_GRAPH="$(mktemp /tmp/benchmark_graph_XXXX.html)"
-    pipenv run python3 graph.py "${CSV_OUTPUT}" "${PLOT_METRIC}" --charts_output="${BENCHMARK_GRAPH}"
-    dt=$(date +'%Y%m%d-%H')
-    RELEASE="$(cut -d'/' -f3 <<<"${CB_GCS_FULL_STAGING_PATH}")"
-    GRAPH_NAME="${RELEASE}.${dt}.${PLOT_METRIC}"
-    gsutil -q cp "${BENCHMARK_GRAPH}" "gs://$CB_GCS_BUILD_PATH/${GRAPH_NAME}"
-  fi
+function generate_graph() {
+  local PLOT_METRIC=$1
+  BENCHMARK_GRAPH="$(mktemp /tmp/benchmark_graph_XXXX.html)"
```

Comment:
I think we'd better convert this .html to .png then .svg. I would like to dump .svg file.

Reply:
is it because we need to manipulate the plots with Javascript? and we want .svg or .png, bokeh can export to both
if that's the case, we need to update the graph.py code to add different export option

Reply:
JS needs data from .csv file. I want .png and .svg, because it's better for me to organize those in files for people to download on the istio-perf website. Could you add the .png and .svg option when you get a time? Thank you

Reply:
added